RxCocoa/Common/RxCocoa.swift

```diff
@@ -287,6 +287,15 @@ func castOrFatalError<T>(value: AnyObject!) -> T {
     return result
 }
 
+func castOrFatalError<T>(value: Any!) -> T {
+    let maybeResult: T? = value as? T
+    guard let result = maybeResult else {
+        rxFatalError("Failure converting from \(value) to \(T.self)")
+    }
+    
+    return result
+}
+
```

Comment:
Yeah, I think you can just change from `func castOrFatalError<T>(value: AnyObject!) -> T {` to `func castOrFatalError<T>(value: Any!) -> T {`

Currently don't see any reason why we would need two functions.